etc/testing/travis_install.sh

```diff
@@ -2,6 +2,10 @@
 
 set -ex
 
+# TODO(lukemarsden): try bumping dist: to bionic in .travis.yml and setting
+# services: - docker to see if docker 18.06 is sufficient, rather than
+# downloading docker every time. Also, why do we use devicemapper?
```

Comment:
There was an attempt to upgrade dist to bionic, along with upgrading minikube [here.](https://github.com/pachyderm/pachyderm/pull/4799) It ended up being less straight-forward that I thought, so I scrapped it. Just upgrading the dist to bionic (w/o upgrading minikube) may work though.

Reply:
Thanks! I'll probably just delete this comment then. I was thinking of ways to speed up the setup, and not having to download and install docker every time would have helped a bit, but it's probably not the overriding cost.